runtime/ms-rest-azure/lib/login.js

```diff
@@ -9,7 +9,7 @@ const ApplicationTokenCredentials = require('./credentials/applicationTokenCrede
 const DeviceTokenCredentials = require('./credentials/deviceTokenCredentials');
 const UserTokenCredentials = require('./credentials/userTokenCredentials');
 const AzureEnvironment = require('./azureEnvironment');
-const SubscriptionClient = require('azure-arm-resource').SubscriptionClient;
+const SubscriptionClient = require('./subscriptionManagement/subscriptionClient');
```

Comment:
made the subscription client local. So no more circular dependency on azure-arm-resource